src/Business Foundation/App/NoSeries/src/Single/NoSeriesImpl.Codeunit.al

```diff
@@ -52,13 +55,31 @@ codeunit 304 "No. Series - Impl."
         TestManualInternal(NoSeriesCode, StrSubstNo(PostErr, DocumentNo));
     end;
 
-    local procedure TestManualInternal(NoSeriesCode: Code[20]; ErrorText: Text);
+    local procedure TestManualInternal(NoSeriesCode: Code[20]; ErrorText: Text)
     var
         NoSeries: Record "No. Series";
     begin
         NoSeries.Get(NoSeriesCode);
         if not NoSeries."Manual Nos." then
-            Error(ErrorText);
+            if HandleActionableManualNosError(NoSeries) then
+                if not NoSeries.Get(NoSeriesCode) and not NoSeries."Manual Nos." then
+                    Error(ErrorText);
```

Comment:
```suggestion
            if not HandleActionableManualNosError(NoSeries) then               
                Error(ErrorText);
```

See also 

As I understand, `HandleActionableManualNosError` returns `true` as soon as "Manual Nos." is already true or is confirmed by the user to set it as such.
I also don't see why we need a `NoSeries.Get` again in line 65 after it has been already fetched in line 62?

See also line 344.

Reply:
Needed the Get since we want to confirm the Manual Nos. has been set, but I can set it as var instead